README.md

```diff
@@ -1,42 +1,40 @@
 # operator-workshop
 
-## Create PostgreSQL deployment
+## Setup
 
-```bash
-$ kubectl apply -f postgresql.yaml
-```
+During the workshop the operator we create will manage PostgreSQL databases.
+A Kubernetes deployment and service is provided to run PostgreSQL in Minikube.
 
-## Run postgresops
-
-```bash
-$ go run main.go
-```
+## Create PostgreSQL database
 
-## Delete PostgreSQL deployment
+Create the k8s resources.
 
 ```bash
-$ kubectl delete -f postgresql.yaml
+$ kubectl apply -f postgresql.yaml
 ```
 
-## Connect to database (optional)
+## List databases
 
-Set password env var from secret.
+List the PostgreSQL databases and their owners.
 
 ```bash
-$ PGPASSWORD=$(kubectl get secret workshop-postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode; echo)
+$ kubectl exec  \
+  $(kubectl get pod -l 'app=workshop-postgresql' -o jsonpath='{.items[0].metadata.name}') \
+  -- psql -U postgres postgres -c "\list"
```

Comment:
I needed to use jsonpath to get the pod name. `\list` is the standard way to list databases and has the owners.

Example output

```
                                 List of databases
   Name    |  Owner   | Encoding |  Collate   |   Ctype    |   Access privileges
-----------+----------+----------+------------+------------+-----------------------
 postgres  | postgres | UTF8     | en_US.utf8 | en_US.utf8 |
 template0 | postgres | UTF8     | en_US.utf8 | en_US.utf8 | =c/postgres          +
           |          |          |            |            | postgres=CTc/postgres
 template1 | postgres | UTF8     | en_US.utf8 | en_US.utf8 | =c/postgres          +
           |          |          |            |            | postgres=CTc/postgres
(3 rows)
```